src/parts/events.js

```diff
@@ -449,7 +463,7 @@ export default {
                 }
 
                 case 'Enter' :
-                    if( this.state.dropdown.visible || e.keyCode == 229 ) return
+                    if( this.state.dropdown.visible ) return
```

Comment:
#539 introduce keycode 229 ignorance. But this is not standard way to solve IME composition problem.

Composition events are defined in the stardard.
https://developer.mozilla.org/en-US/docs/Web/API/Element/compositionstart_event